tools/cockpit.spec

```diff
@@ -253,6 +262,11 @@ rm -rf %{buildroot}/debug
 %{_datadir}/dbus-1/services/com.redhat.Cockpit.service
 %{_libexecdir}/cockpitd
 
+%files udisks
+%{_libexecdir}/cockpit-udisks
+%{_datadir}/dbus-1/services/com.redhat.Cockpit.Storage.service
+%doc %{_mandir}/man8/cockpit-udisks.8.gz
```

Comment:
I think this is an implementation detail, and doesn't need a man page. Similar to how 'cockpit-pcp' doesn't have a man page.